sdk/ai/azure-ai-resources/azure/ai/resources/_asset_client/_client.py

```diff
@@ -0,0 +1,164 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+import re
+from typing import Optional
+
+from azureml._base_sdk_common.service_discovery import get_service_url
+from azureml.core import Run, Workspace
+from azureml.exceptions import UserErrorException
+from azure.ai.generative.index._utils.logging import get_logger
```

Comment:
we shouldn't refer to azure-ai-generative from ai-resources. that's another reason i think maybe we should move it to generative.

Reply:
My bad. azure-ai-resources has its own logging as well.